frontend/src/app/components/datetime/timezone.service.spec.ts

```diff
@@ -59,7 +59,7 @@ describe('TimezoneService', function () {
       ]
     });
 
-    timezoneService = TestBed.get(TimezoneService);
+    timezoneService = TestBed.inject(TimezoneService);
```

Comment:
Drive-by fix 🚜: this (and following similar changes) are fixing a deprecation warning I spotted while reading code